src/components/Tooltip/tests/Tooltip.test.tsx

```diff
@@ -31,8 +31,11 @@ describe('<Tooltip />', () => {
     );
   });
 
-  it('renders on mouseEnter', () => {
-    wrapperComponent.simulate('mouseEnter');
+  it('renders on mouseOver', () => {
+    wrapperComponent.simulate('mouseOver');
```

Comment:
Do we need the extra simulate and expect?

Reply:
Just saw the commit message was for codecov. Maybe we should break the assertion into its own test and expect mouseOver is called once?

Reply:
Ya, I was testing to see if it would satisfy codecov, which it did. It definitely feels hacky, but I'm not sure how else to satisfy codecov. I think ideally we'd assert that `handleMouseEnter` is only called once, but it's a private method.

>Maybe we should break the assertion into its own test and expect mouseOver is called once?

I don't follow. Can you elaborate?

Reply:
Ignore that, I was under the assumption we exposed a mouseEnter handler 😅 These changes may give you codecov
```tsx
  private handleMouseEnter = () => {
    this.mouseEntered = true;
    this.setState({active: true});
  };

  private handleMouseEnterFix = () => {
    !this.mouseEntered && this.handleMouseEnter();
  };
```

Reply:
Nice! I'll give that a shot. :+1: